azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -494,20 +494,24 @@ VersionSet fetchVersionSet(final int executionId, final Map<String, String> flow
             if (flowParams.containsKey(FlowParameters.FLOW_PARAM_ALLOW_IMAGE_TEST_VERSION) &&
                 Boolean.TRUE.equals(Boolean
                     .valueOf(flowParams.get(FlowParameters.FLOW_PARAM_ALLOW_IMAGE_TEST_VERSION)))) {
-              versionMap.put(imageType,
+              overlayMap.put(imageType,
```

Comment:
Can you please add an info log in this if and else case?

Reply:
added info logs.